backend/src/core/config_manager.py

```
@@ -1,15 +1,30 @@
+    #配置总线应该不能有好几个吧。。
+    _instance = None
+    _initialized = False
+
+    def __new__(cls):
+        if cls._instance is None:
+            cls._instance = super().__new__(cls)
+        return cls._instance
+    
     def __init__(self):
+        if hasattr(self, '_initialized_attr'):
+            log.error(f"Event bus has already been initialized !")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)